app/assets/javascripts/discourse/tests/integration/components/user-menu/menu-test.js

```diff
@@ -86,7 +86,39 @@ module("Integration | Component | user-menu", function (hooks) {
     );
   });
 
+  test("reviewables tab is shown if currentUser can review", async function (assert) {
+    this.currentUser.set("can_review", true);
+    await render(template);
+    const tab = query("#user-menu-button-review-queue");
+    assert.strictEqual(tab.dataset.tabNumber, "4");
+
+    const tabs = Array.from(queryAll(".tabs-list .btn")); // top and bottom tabs
+    assert.strictEqual(tabs.length, 6);
+
+    assert.deepEqual(
+      tabs.map((t) => t.dataset.tabNumber),
+      [...Array(6).keys()].map((n) => n.toString()),
```

Comment:
```suggestion
      ['0', '1', '2', '3', '4', '5'],
```

I think this is way easier to read.